src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package movie.storage.service.impl;
+
+import java.util.ArrayList;
+import movie.storage.dao.ShoppingCartDao;
+import movie.storage.dao.TicketDao;
+import movie.storage.lib.Inject;
+import movie.storage.lib.Service;
+import movie.storage.model.MovieSession;
+import movie.storage.model.ShoppingCart;
+import movie.storage.model.Ticket;
+import movie.storage.model.User;
+import movie.storage.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        ShoppingCart shoppingCart = shoppingCartDao.getByUser(user);
+        Ticket ticket = new Ticket();
+        ticket.setUser(user);
+        ticket.setMovieSession(movieSession);
+        Ticket ticketDB = ticketDao.add(ticket);
```

Comment:
```suggestion
        Ticket ticketDb = ticketDao.add(ticket);
```

or you can continue to use `ticket`